lighthouse-core/config/config-helpers.js

```diff
@@ -139,7 +139,9 @@ function requireAudits(audits, configDir) {
       let requirePath = `../audits/${audit.path}`;
       if (!coreAudit) {
         // Otherwise, attempt to find it elsewhere. This throws if not found.
-        requirePath = resolveModule(audit.path, configDir, 'audit');
+        const absolutePath = resolveModule(audit.path, configDir, 'audit');
+        // Use a relative path so bundler can easily expose it.
+        requirePath = path.relative(__dirname, absolutePath);
```

Comment:
needs to be relative, otherwise we'd need to be browserify exposing an absolute path

Reply:
I don't totally understand the comment and this line. Isn't the bundler exposing the pub audits by manually requiring in its config file?

It seems like this path of requireAudits is only hit when we're requiring in the audits as we execute the bundle where we're in brfs land and don't have real dirnames anymore. Is that right?

Reply:
> It seems like this path of requireAudits is only hit when we're requiring in the audits as we execute the bundle where we're in brfs land and don't have real dirnames anymore. Is that right?

right. Internal to the bundle they have some module name that can be required. Since they're required dynamically, the name they're exposed as has to be figured out at runtime, and the easiest way to do that is to use the same name that is passed to `require()` in non-browserified execution. The existing code was doing that with an absolute path here, and even though we could expose the modules with the same absolute path in `build-bundle`, those are going to be different based on what machine it's run on, etc. Hence using a relative path here, which also works non-browserified.

It's basically the dynamic `require.resolve()` equivalent of the `../audits/${audit.path}` for the core audits a few lines above this.